File: spring-boot-emberjs-backend/src/main/java/com/models/Post.java
package com.models;

import lombok.Data;

import javax.persistence.*;
import java.sql.Date;

/**
 * @author Roman Nagibov
 */
@Data

@Entity
@Table(name = "posts")
public class Post {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "title")
    private String title;

    @Column(name = "date")
    private Date date;
    
    @Column(name = "body")
    private String body;

    @ManyToOne
    @JoinColumn(name = "author_id")
    private Author author_id;
}

File: spring-boot-emberjs-frontend/app/models/post.js
import Ember from 'ember';
import DS from "ember-data";
import { Model } from 'ember-pouch';

var Post = Model.extend({
  title: DS.attr('string', {defaultValue: ""}),
  author: DS.belongsTo('author'),
  date: DS.attr('date'),
  body: DS.attr('string', {defaultValue: ""}),

  authorName: Ember.computed.readOnly('author.name')
});

export default Post;
